.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "submodules/expected"]
+	path = submodules/expected
```

Comment:
Usually, the folder is named `deps` or `dependecy` or `3rdparty`.
`Submodule` can be some extension, but the name does not sound like some a 3rd party dependency.